### PR TITLE
infra: Hotfix for missing lorax-templates-rhel

### PR DESCRIPTION
**Pull request by jkonecny12**
We need latests version of this package to make ISO build working. Let's install that from the koji directly.

### CODE REVIEW

File: dockerfile/anaconda-iso-creator/Dockerfile [hunk 1]
```diff
@@ -38,6 +38,9 @@ RUN set -ex; \
 # required for lorax-build-webui script
   patch \
   lorax; \
+  if grep -q 'ID=.*centos' /etc/os-release && grep -q 'VERSION_ID=.*10' /etc/os-release ; then \
```
Comment (KKoukiou) [inline dockerfile/anaconda-iso-creator/Dockerfile:41]: Please specify which upstream release this is required and add a FIXME comment.

https://gitlab.com/redhat/centos-stream/rpms/lorax-templates-rhel/-/blob/c9s/lorax-templates-rhel.spec?ref_type=heads#L3 c10s has 39

Can you find which upstream tag we need to be synced? https://github.com/weldr/lorax/tags

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

**Review by KKoukiou — APPROVED** [review]

**Pull request closed without merging**